pkg/jx/cmd/install.go

```diff
@@ -1630,58 +1638,70 @@ func (options *InstallOptions) generateGitOpsDevEnvironmentConfig(gitOpsDir stri
 			}
 			repo, gitProvider, err := kube.CreateEnvGitRepository(options.BatchMode, authConfigSvc, devEnv, devEnv, config, forkEnvGitURL, envDir,
 				gitRepoOptions, options.CreateEnvOptions.HelmValuesConfig, prefix, git, options.ResolveChartMuseumURL, options.In, options.Out, options.Err)
 			if err != nil || repo == nil || gitProvider == nil {
-				return errors.Wrap(err, "creating git repository for the dev environment source")
+				return "", errors.Wrap(err, "creating git repository for the dev environment source")
 			}
 
 			dir := gitOpsDir
 			err = git.Init(dir)
 			if err != nil {
-				return errors.Wrap(err, "initializing the dev environment repository")
+				return "", errors.Wrap(err, "initializing the dev environment repository")
 			}
 			err = options.ModifyDevEnvironment(func(env *v1.Environment) error {
 				env.Spec.Source.URL = repo.CloneURL
 				env.Spec.Source.Ref = "master"
 				return nil
 			})
 			if err != nil {
-				return errors.Wrap(err, "updating the source in the dev environment")
+				return "", errors.Wrap(err, "updating the source in the dev environment")
 			}
 
 			err = git.Add(dir, ".gitignore")
 			if err != nil {
-				return errors.Wrap(err, "adding gitignore to the dev environemnt")
+				return "", errors.Wrap(err, "adding gitignore to the dev environemnt")
 			}
 			err = git.Add(dir, "*")
 			if err != nil {
-				return errors.Wrap(err, "adding all files from dev environment repo to git")
+				return "", errors.Wrap(err, "adding all files from dev environment repo to git")
 			}
 			err = options.Git().CommitIfChanges(dir, "Initial import of Dev Environment source")
 			if err != nil {
-				return errors.Wrap(err, "committing in git if there are changes")
+				return "", errors.Wrap(err, "committing in git if there are changes")
 			}
 			userAuth := gitProvider.UserAuth()
 			pushGitURL, err := git.CreatePushURL(repo.CloneURL, &userAuth)
 			if err != nil {
-				return errors.Wrapf(err, "creating push URL for %q", repo.CloneURL)
+				return "", errors.Wrapf(err, "creating push URL for %q", repo.CloneURL)
 			}
 			err = git.SetRemoteURL(dir, "origin", pushGitURL)
 			if err != nil {
-				return errors.Wrapf(err, "setting remote origin to %q", pushGitURL)
+				return "", errors.Wrapf(err, "setting remote origin to %q", pushGitURL)
 			}
 			err = git.PushMaster(dir)
 			if err != nil {
-				return errors.Wrapf(err, "pushing master from repository %q", dir)
+				return "", errors.Wrapf(err, "pushing master from repository %q", dir)
 			}
 			log.Infof("Pushed Git repository to %s\n\n", util.ColorInfo(repo.HTMLURL))
+
+			dir = filepath.Join(envDir, gitRepoOptions.Owner)
+			if _, err := os.Stat(dir); os.IsNotExist(err) {
+				if err := os.MkdirAll(dir, 0755); err != nil {
+					return "", errors.Wrapf(err, "creating directory %q", dir)
+				}
+			}
+			dir = filepath.Join(dir, repo.Name)
+			if err := os.Rename(gitOpsDir, dir); err != nil {
```

Comment:
@ccojocar in the integration tests you seem to be running into an equivalent of this https://github.com/jenkins-x/jx/issues/449

Reply:
It seems so. I am trying to use the utility method instead.